--- a/lib/NOM/IO.hs
+++ b/lib/NOM/IO.hs
@@ -89,7 +89,7 @@ writeStateToScreen pad printed_lines_var nom_state_var nix_output_buffer_var ref
     pure (nom_state, nix_output_raw)
   -- ====
 
-  let nix_output = ByteString.lines $ ByteString.concat $ reverse nix_output_raw
+  let nix_output = []
       nix_output_length = length nix_output
 
       nom_output = ByteString.lines = $ encodeUtf8 $ truncateOutput terminalSize (printer nom_state terminalSize (nowClock, now))
